tests/skylab/smartseq2_multisample/pr/ValidateMultiSampleSmartSeq2.wdl

```diff
@@ -18,7 +18,7 @@ task ValidateSmartSeq2Plate {
   >>>
   
   runtime {
-    docker: "quay.io/humancellatlas/secondary-analysis-loom-output:0.0.2"
+    docker: "quay.io/humancellatlas/secondary-analysis-loom-output:0.0.3-fk-2"
```

Comment:
This seems like a very unofficial name - should this have the initials in it? Why isn't it a released version of the docker image?